AppCenterCrashes/AppCenterCrashesTests/MSWrapperCrashesHelperTests.mm

```diff
@@ -0,0 +1,269 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "MSAppCenterInternal.h"
+#import "MSChannelGroupProtocol.h"
+#import "MSChannelUnitConfiguration.h"
+#import "MSChannelUnitProtocol.h"
+#import "MSCrashesInternal.h"
+#import "MSCrashesPrivate.h"
+#import "MSCrashesTestUtil.h"
+#import "MSCrashesUtil.h"
+#import "MSErrorAttachmentLog.h"
+#import "MSException.h"
+#import "MSHandledErrorLog.h"
+#import "MSLogWithProperties.h"
+#import "MSTestFrameworks.h"
+#import "MSWrapperCrashesHelper.h"
+
+@interface MSWrapperCrashesHelperTests : XCTestCase
+@end
+
+static NSString *const kMSTestAppSecret = @"TestAppSecret";
+static NSString *const kMSTypeHandledError = @"handledError";
+
+@implementation MSWrapperCrashesHelperTests
```

Comment:
It doesn't look like there are any tests that try to send more than one attachment of each type

Reply:
Are you talking about handled error with one text attachment and one binary attachment? https://github.com/microsoft/appcenter-sdk-apple/pull/1853/files#diff-9434b2604d9fd78f2f68080318217227R256 does that already.

Reply:
@jwallra There aren't any actual code paths that that would matter. It's just in the portal that it only shows 1 text and 1 binary. The SDK will happily send as many as they want. We are covered with sending at least 2 in a test, and at least 1 of each type